pkg/render/fluentd.go

```diff
@@ -358,6 +362,45 @@ func (c *fluentdComponent) fluentdServiceAccount() *corev1.ServiceAccount {
 	}
 }
 
+func (c *fluentdComponent) podExecRole() *rbacv1.Role {
```

Comment:
Definitely need a comment explaining why this pod/exec role is here and what it is used for.